API.md

```diff
@@ -1137,17 +1137,17 @@ mapboxClient.getDirections([
 
 Returns **Promise** response
 
-## getDistances
+## getMatrix
 
 Compute a table of travel-time estimates between a set of waypoints.
-Consult the [Mapbox Distance API](https://www.mapbox.com/developers/api/distance/)
+Consult the [Mapbox Matrix API](https://www.mapbox.com/api-documentation/#matrix)
 for more documentation.
 
 **Parameters**
 
--   `waypoints` **Array&lt;Array&lt;number&gt;&gt;** an array of coordinate pairs
+-   `waypoints` **Object&lt;Object&lt;number&gt;&gt;** an array of coordinate object pairs
     in [longitude, latitude] order. Up to
-    100 waypoints can be specified.
+    25 waypoints can be specified.
```

Comment:
Hmm, to avoid this number going stale, should we just refer to the online docs for "coordinate limits" ?  We may change/increase those at any time, it'd be nice not to need doc updates here when/if there are limit changes.